com.unity.shadergraph/Documentation~/surface-options.md

```diff
@@ -0,0 +1,38 @@
+# Modify surface options without changing your graph
+
+## Description
+
+Enable **Allow Material Override** to modify a specific set of properties for Universal Render Pipeline Lit and Unlit Shader Graphs and for Built-In Render Pipeline Shader Graphs in the Material Inspector.
+
+This functionality makes it possible for you to override the following Universal Render Pipeline Lit shader properties:
+
+* Workflow Mode
+* Surface Type
+* Render Face
+* Depth Write
+* Depth Test
+* Alpha Clipping
+* Cast Shadows
+* Receive Shadows
+
+These are the Universal Render Pipeline Unlit shader properties you can override:
+
+* Surface Type
+* Render Face
+* Depth Write
+* Depth Test
+* Alpha Clipping
+* Cast Shadows
+
+The Universal Render Pipeline documentation for [Unlit](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@11.0/manual/unlit-shader.html) and [Lit](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@11.0/manual/lit-shader.html) shaders explains these properties, which correspond to specific [ShaderLab](https://docs.unity3d.com/Manual/SL-Reference.html) tags and commands that are also applicable to the Built-In Render Pipeline.
```

Comment:
", which correspond to specific [ShaderLab](https://docs.unity3d.com/Manual/SL-Reference.html) tags and commands that are also applicable to the Built-In Render Pipeline." - I would remove this, as it's too vague to be helpful. 

Updated to add: In fact, I would remove this whole para in favour of changing these lists to tables, and providing links for each property. See above!

Reply:
Fixed